receiver/purefbreceiver/go.mod

```
@@ -0,0 +1,182 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/purefbreceiver
+
+go 1.18
+
+require (
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.69.0
```

There was a new release this morning, this should be:

```suggestion
	github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.70.0
```